src/riscv/i_system.c

```diff
@@ -39,6 +39,21 @@
 
 #include "console.h"
 
+#ifdef COMBINE_SCREENS
+unsigned char CombinedScreens[SCREENWIDTH*SCREENHEIGHT];
```

Comment:
Mark `static` modifier with `CombinedScreens` if you confirm its visibility. It aids compiler optimizations.